packages/components/package.json

```diff
@@ -76,6 +76,7 @@
         "@deriv/shared": "^1.0.0",
         "@deriv/stores": "^1.0.0",
         "@deriv/translations": "^1.0.0",
+        "@deriv/hooks": "^1.0.0",
```

Comment:
any reason why you are adding hooks package?

Reply:
Will remove this 🙏